### PR TITLE
Adds some new grindables to the grinder

### PR DESCRIPTION
**Pull request by Anonmare**
[Changelogs]: Adds several new items that the grinder can grind, for the aspiring ghetto chemist

:cl: 
add: Adds new grindables
/:cl:

[why]: I'm currently looking into adding ghetto chemistry for aspiring assistants (both loyal and disloyal) to make use of in their endeavours, this PR adds new things that can be scavenged and ground up for those without chemistry machines. If anyone wants to make any suggestions or give pointers, I'd be more than happy to hear them

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Super cool

**Comment by RemieRichards** [comment]
At this point all these lists should be on the individual types, not like this anymore.

**Comment by nicbn** [comment]
Cable is not made of copper is it?

**Comment by Anonmare** [comment]
@nicbn Technically no as far as the lathe is concerned but I'm willing to take some liberties with a few things that would be "reasonable"

File: code/modules/reagents/chemistry/machinery/reagentgrinder.dm [hunk 1]
```diff
@@ -50,12 +50,21 @@
...
+			/obj/item/photo = list ("iodine" = 4),
+			/obj/item/pen = list ("iodine" = 2, "iron" = 1),
+			/obj/item/reagent_containers/food/drinks/soda_cans = list ("aluminium" = 10),
 			//All types that you can put into the grinder to transfer the reagents to the beaker. !Put all recipes above this.!
 			/obj/item/slime_extract = list(),
 			/obj/item/reagent_containers/pill = list(),
 			/obj/item/reagent_containers/food = list(),
 			/obj/item/reagent_containers/honeycomb = list(),
 			/obj/item/toy/crayon = list())
+
```
Comment (DoctoreMad) [inline code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: Nice space

**Comment by DoctoreMad** [comment]
Specify changelog a bit, maybe?

add: More items can now be used in a all-in-one grinder to get ghetto chemistry components.

**Comment by Anonmare** [comment]
@docotremad I'm working on adding a mortar and pestle craftable so that you don't have to rely on the all-in-one grinder only. Also working on making a craftable electrolyzer and a primitive chemmaster that can only separate chemicals

File: code/modules/reagents/chemistry/machinery/reagentgrinder.dm (hunk 1 above)
Comment (Anonmare) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: Oh darn I thought I got that already

**Comment by DoctoreMad** [comment]
Basically /vg/station ghetto chem then?

**Comment by Anonmare** [comment]
I guess so, perhaps it might be worth looking into how they've managed it  and seeing if I can learn something

**Comment by Anonmare** [comment]
Also if anyone has any ideas for anything that would, reasonably, contain mercury and/or radium I'd like to hear it. Preferably something common and easy to get

**Comment by Bawhoppen** [comment]
Another ghetto chem tip: You can get fluorine from bottled water

**Comment by YakumoChen** [comment]
@anonmare put mercury in analyzers, since they measure temperature. Maybe radium in medical analyzers or Geiger counters since they measure radiation? I feel like this somehow makes sense.

**Comment by bgobandit** [comment]
soap => lye

**Comment by Bawhoppen** [comment]
Issue is, some of these only take metal to make them, thus creating infinite chems from nothing.

Solution: Add chem requirements to autolathe

**Comment by ACCount12** [comment]
Not an issue, chemistry has a much easier way to create infinite chems from nothing. Sure, the dispenser may run out of power, but your average Assistant McChemist will run out of patience even faster.

File: code/modules/reagents/chemistry/machinery/reagentgrinder.dm [hunk 2]
```diff
@@ -50,6 +50,18 @@
 			/obj/item/reagent_containers/food/snacks/grown/coffee = list("coffeepowder" = 0),
 			/obj/item/reagent_containers/food/snacks/grown/tea/astra = list("teapowder" = 0, "salglu_solution" = 0),
 			/obj/item/reagent_containers/food/snacks/grown/tea = list("teapowder" = 0),
+			//Stuff that doesn't quite fit in the other categories
+			/obj/structure/cable = list ("copper" = 5),
```
Comment (ACCount12) [inline code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: Not going to work. This is object cable, not item cable.

**Comment by Bawhoppen** [comment]
Just because of balance not being involved, doesn't mean it's not a huge consistency issue

**Comment by ACCount12** [comment]
what is consistency

**Comment by Bawhoppen** [comment]
It being consistent in having the same materials that go in, come out

**Comment by YakumoChen** [comment]
adding chem requirements to autolathe would both ruin the point of trying to avoid using chemistry and would make most things unprintable because chemistry sure as hell isn't going to provide chems all day for it

**Comment by Bawhoppen** [comment]
We need less watered down crap anyways actually

**Comment by Anonmare** [comment]
@YakumoChen Don't know how I feel about geiger counters, will have to come back to that suggestion. Gas analyzers will now have a small amount of mercury in them, since Nanotrasen is cheap enough to put dangerous chemicals in their commonly used equipment.

**Comment by pantherpusher** [comment]
What if you got radium from glowsticks

**Comment by davethwave** [comment]
Radium in dna injectors then?

**Comment by davethwave** [comment]
Considering they change the dna of a human momentarily they use controlled radiation and thus radium

**Comment by davethwave** [comment]
lighters and stuff for welder fuel, the more fancy it is the more welding fuel it has in it. Flare for phosphorus. Power cell/capacitor for iodine and sulfuric acid. Welding tool for iron/welding fuel(depends on how full the welding tool is). Night vision goggles as stuff like it can give uranium or some other precious metals like gold or silver as they are created with them. Monkey cubes for blood/nutriment perhaps?

**Comment by Supermichael777** [comment]
welder-fuel is common if cargo aren't being thieving assholes.

so yeah an alt source would be nice

**Comment by Anonmare** [comment]
Hmm, I don't entirely know how to have the grinder account for the welding tool's fuel capacity but lighters are easily doable. I'd prefer to keep this list to commonly found/unrestricted station items but I'll try adding some rarer items as well

**Comment by davethwave** [comment]
Also how about grinding up cigarettes for nicotine and some minor drugs, antag tokens may contain a few units of dangerous chems in them like cyanide and the like, bruisepacks will give styptic powder when ground up, and ointment will give silver sulfadiazine when ground up.

**Comment by Anonmare** [comment]
Hmm, I already did cigarettes but I can go for bruise packs and ointment at the very least. Ideally, I want the most common elements of chemistry to be acquirable by assistants in some capacity.

**Comment by davethwave** [comment]
So what are you missing on the common ingredients list then?

**Comment by Anonmare** [comment]
I've covered most of my bases, carbon I'm not overly happy with but I can't think of something that would be a pure source of it

**Comment by davethwave** [comment]
How about the standard power cells? For instance you can get carbon rods out of old car batteries I believe.

File: code/modules/reagents/chemistry/machinery/reagentgrinder.dm [hunk 3]
```diff
@@ -45,17 +46,39 @@
...
+			/obj/item/stack/medical/bruise_pack = list ("styptic_powder" = 5),
+			/obj/item/stack/medical/ointment = list ("silver_sulfadiazine" = 5),
 			//All types that you can put into the grinder to transfer the reagents to the beaker. !Put all recipes above this.!
 			/obj/item/slime_extract = list(),
 			/obj/item/reagent_containers/pill = list(),
 			/obj/item/reagent_containers/food = list(),
 			/obj/item/reagent_containers/honeycomb = list(),
-			/obj/item/toy/crayon = list())
+			/obj/item/toy/crayon = list()),
+			/obj/item/clothing/mask/cigarette = list()
```
Comment (Jordie0608) [inline code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: You need the extra `)` from the line above to be here.

`/obj/item/toy/crayon = list(),`
`/obj/item/clothing/mask/cigarette = list())`

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

**Review by Jordie0608 — APPROVED** [review]

**Comment by davethwave** [comment]
how about grinding electronics and boards for silicon? Also another way to get carbon via real life chems is sulfuric acid and sucrose. https://www.youtube.com/watch?v=XGLuZhBWLrQ

**Comment by davethwave** [comment]
Also supposedly when sugar gets hot enough the sugar turns into carbon as well. https://www.youtube.com/watch?v=9d8oaBHaTwY

**Comment by Anonmare** [comment]
Already get electronics and computer boards, grinding a circuitboard will give you 0.5u of sulphuric acid as well (This is for balance so you can't produce more acid than you use, even with bluespace parts). I'll be working on adding some new things for ghetto chemistry, like electrolysis to breakdown reagents into their constituents but that's outside of this PR for now

**Comment by naltronix** [comment]
FUCK YOU BEAT ME TO IT
FUCKING HELL DAMNIT REEE

**Pull request merged**